src/components/navigationbar.scss

```diff
@@ -65,3 +66,25 @@
     }
   }
 }
+
+@media #{$mobile-version} {
+  .navBarIcon {
+    padding-left: 30px;
+  }
+
+  .navBarItems {
+    order: 3;
+    margin-top: 40px;
+    width: 100%;
+  }
+
+  .navBarLogIn {
+    a {
+      font-size: 12px;
+    }
+
+    button {
+      background-color: red;
```

Comment:
If you are talking about this one not working... it's because the CSS specificity level is higher than the mobile version. You can see it on the Devtools the rule ~~in strikethrough text~~. Do you understand how/why the specificity is lower?

<img width="1209" alt="image" src="https://user-images.githubusercontent.com/14869087/181093780-b2190997-f421-4b76-9a5b-5443b93a5437.png">

Reply:
I just found out why. Thanks for the tip!